consts/consts.go

```diff
@@ -590,3 +591,11 @@ var LATENCY_LOG_OPERATIONS_FOR_SEARCH = []string{
 	LAT_DOSEARCH_MULTISEARCHTWEETSDO,
 	LAT_DOSEARCH_TYPOSUGGESTDO,
 }
+
+const (
+	SRC_SHAMATI = 141
+)
+
+var ES_SRC_PARENTS_FOR_CHAPTER_POSITION_INDEX = map[int64]bool{
+	SRC_SHAMATI: true,
```

Comment:
Only for Shamati?
Nothing else?

Reply:
Search for Shamati was the major problem. Later we will consider adding more sources.